src/components/Editor.jsx

```diff
@@ -130,6 +130,13 @@ class Editor extends React.Component {
           this.props.onAutoFormat();
         },
       });
+      this._editor.commands.addCommand({
```

Comment:
An `addCommands` method would be nice here to combine with the above but doesn't look like that is an option.